src/password-store.sh

```diff
@@ -17,14 +17,30 @@ X_SELECTION="${PASSWORD_STORE_X_SELECTION:-clipboard}"
 CLIP_TIME="${PASSWORD_STORE_CLIP_TIME:-45}"
 GENERATED_LENGTH="${PASSWORD_STORE_GENERATED_LENGTH:-25}"
 
-export GIT_DIR="${PASSWORD_STORE_GIT:-$PREFIX}/.git"
 export GIT_WORK_TREE="${PASSWORD_STORE_GIT:-$PREFIX}"
+export GIT_DIR="$GIT_WORK_TREE/.git"
 
 #
 # BEGIN helper functions
 #
-
+git_set_dir() {
+	local gwt="${PASSWORD_STORE_GIT:-$PREFIX}"; gwt="${gwt%/}"
```

Comment:
Yeah I got burned in a few test situations with it being overwritten IIRC, so opted to be explicit in the local scope here.